testhelpers/docker.go

```diff
@@ -54,15 +49,34 @@ func DockerRun(t *testing.T, image string, ops ...DockerCmdOp) string {
 	return Run(t, exec.Command("docker", append([]string{"run", "--rm"}, args...)...))
 }
 
-func DockerRunAndCopy(t *testing.T, containerName, copyDir, image, path string, ops ...DockerCmdOp) string {
+//DockerRunAndCopy runs a container and once stopped, outputCtrPath is copied to outputDir
+func DockerRunAndCopy(t *testing.T, containerName, outputDir, outputCtrPath, image string, ops ...DockerCmdOp) string {
 	ops = append(ops, WithFlags("--name", containerName))
 	args := formatArgs([]string{image}, ops...)
 
 	output := Run(t, exec.Command("docker", append([]string{"run"}, args...)...))
-	Run(t, exec.Command("docker", "cp", containerName+":"+path, copyDir))
+	Run(t, exec.Command("docker", "cp", containerName+":"+outputCtrPath, outputDir))
 	return output
 }
 
+//DockerSeedRunAndCopy copies srcDir to container's srcCtrPath before container is started. Once stopped, outputCtrPath is copied to outputDir
+//Only works for WCOW when seeding to container directory (not a mounted volume)
+func DockerSeedRunAndCopy(t *testing.T, containerName, srcDir, srcCtrPath, outputDir, outputCtrPath, image string, ops ...DockerCmdOp) string {
+	ops = append(ops, WithFlags("--name", containerName))
+	args := formatArgs([]string{image}, ops...)
+
+	output := Run(t, exec.Command("docker", append([]string{"create"}, args...)...))
+	output += Run(t, exec.Command("docker", "cp", srcDir, containerName+":"+srcCtrPath))
+	output += Run(t, exec.Command("docker", "start", "--attach", containerName))
```

Comment:
This is the line that does not work with docker CLI 20.10.0 but works in every other version of Docker, hence the downgrade in the GHA workflow. I attempted to rewrite this to avoid this syntax but couldn't find a simple, cross-platform implementation.

Reply:
Worked around this by using 20.10.5 in GHA